doc/getting_started.md

```diff
@@ -0,0 +1,52 @@
+# Seesaw Getting Started Guide
+
+## About
+
+Seesaw is a load balancer implementation based on the Linux Virtual Server
+project.  As such, it's different from the load balancers many individuals
+are used to.  Unlike solutions like HAProxy or Nginx which can operate all the
+way up to layer seven, Seesaw operates as a layer four load balancer.  This
+means that while it can handle UDP, TCP, SCTP, AH, & ESP traffic, it does not
+go far enough up the OSI stack to handle features like HTTP header
+introspection, TLS termination etc.
+
+## Requirements
+
+In order to operate, Seesaw expects a few kernel modules to be loaded:
+
+  * `ip_vs` - IP Virtual Server module. Used to handle transport layer
+switching within the Linux kernel.
+  * `nf_conntrack_ipv4` - Used by the Linux kernel to maintain the state of all
+logical network connections and relate packets to the sessions.
+  * `dummy` - A virtual network device which can have routes assigned but will
+transfer no packets.
+
+Each of these kernel modules should be loaded before the Seesaw process is
+started.  Most users will find this easiest to achieve through the use of the
+`/etc/modules.load.d` mechanism.  To load each of these automatically with the
+built in `systemd` mechanisms run the following commands:
```

Comment:
Quite possibly.  I hadn't even gotten to the point of worrying about files for the init system yet.  Once I get this a bit further along I'm happy to contribute systemd service unit files.  As for the usage of `/etc/modules.load.d` that should be valid for most distros with the notable exception of Gentoo w/o systemd and Slackware.  Even where Ubuntu calls `rc.local` it's currently done via systemd IIRC.  Regardless, not trying to start a religious war. I'll throw in some clarifying comments about manually running `modprobe` too.